resources/public/api/conf/6.0/examples/adjustments/def3/submit_foreign_property_accounting_adjustments.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
The package structure for def1, def2 and def3 needs to mirror that used for the schemas. We cannot really have all business types lumped together because one business type might have def1 and def2 whereas another might only have def1.

Also in this case it looks like only def1 and def3 examples are being used. Maybe (??) remove def2 altogether and rename def1 to def1_and_2 or something. Not sure.